package.json

```diff
@@ -57,7 +57,9 @@
   "homepage": "https://github.com/webex/react-ciscospark#readme",
   "dependencies": {
     "@ciscospark/common": "1.49.3",
+    "@ciscospark/common-timers": "1.49.3",
     "@ciscospark/helper-html": "1.49.3",
+    "@ciscospark/http-core": "1.49.3",
```

Comment:
Could these work in devDependencies instead?

Reply:
I don't think so because tests were failing.